config/routes.rb

```diff
@@ -2,22 +2,6 @@
 
 Rails.application.routes.draw do
   get "legal/index"
-  
-  resources :market_places
-  root 'main#index'
```

Comment:
This command is repeated on line 22, At the time of doing the pull of the master it caused conflict.

Reply:
@KarinaDell @danielasotomayor99 @EduJimenez96 Can you help us on this?